lang/ui.ca.json

```diff
@@ -0,0 +1,1161 @@
+{
+  "about": {
+    "defaultMessage": "Quant a",
+    "description": "About action"
+  },
+  "about-comic": {
+    "defaultMessage": "Còmic de tres panells titulat \"MicroPython Rocks\" de Mike Rowbit. Una serp de dibuixos animats presenta a Damien, dient \"Et presento Damien... Va crear MicroPython\". Dues serps parlen de MicroPython. La serp groga diu \"MicroPython està dissenyat per funcionar en ordinadors molt petits\". \"Com la teva micro:bit de la BBC\" respon la serp morada.\" La serp groga continua \"Però Python pot executar-se a qualsevol lloc\". La serp morada hi està d'acord, dient \"Com en aquest bastidor de servidors que fan funcionar llocs de web enormes\". El fons darrere de les serps mostra un bastidor de servidors.",
+    "description": "Alt text for comic in the about dialog"
+  },
+  "about-microbit": {
+    "defaultMessage": "Fet amb amor per la <link>Micro:bit Educational Foundation and contributors</link>",
```

Comment:
Translate text in `<link>`, don't duplicate. In this case just the "and contributors"

Reply:
Requested in Crowdin